server/modules/arenaImport/service/arenaImport/jobs/usersImportJob.js

```diff
@@ -85,6 +86,24 @@ const insertUser = async ({ user, surveyId, survey, arenaSurvey, arenaSurveyFile
   return userAlreadyExisting || user
 }
 
+const insertInvitations = async ({ survey, arenaSurveyFileZip, user, userUuidNewByUserUuid }, client) => {
+  const userInvitations = await ArenaSurveyFileZip.getUserInvitations(arenaSurveyFileZip)
+  if (userInvitations.length > 0) {
+    // exclude invitations with user not in users list (user invited that never accepted the invitation)
+    const userInvitationsValid = []
+    userInvitations.forEach((userInvitation) => {
+      const userUuidNew = userUuidNewByUserUuid[UserInvitation.getUserUuid(userInvitation)]
```

Comment:
maybe is better newUserUuid in both names

Reply:
ok, I'll change it